.github/workflows/node.js.yml

```diff
@@ -6,25 +6,49 @@ name: Node.js CI
 on:
   push:
     branches: [ "main" ]
+    tags:        
+      - '*'
   pull_request:
     branches: [ "main" ]
 
 jobs:
-  build:
+  build-and-submit:
 
     runs-on: ubuntu-latest
-
     strategy:
       matrix:
         node-version: [18.x, 20.x, 22.x]
```

Comment:
Worried matrix will also result in submission happening multiple times.

Reply:
We have confirmed the concurrent submissions.

Thinking about it though, as this is not a node module, we merely use node to produce a compile artefact (two specifically), we don't need to run on anything other than latest node (22 as of today).

Reply:
Removed Matrix completely and only use node 22.x